src/app/client/src/app/modules/notes/components/popup-editor/popup-editor.component.ts

```diff
@@ -162,7 +162,7 @@ export class PopupEditorComponent implements OnInit, AfterViewInit, OnDestroy {
     if (this.ids.contentId || this.ids.courseId) {
       const requestData = {
         request: {
-          note: this.noteData.note,
+          note: $('#wmd-preview').html(),
```

Comment:
why is the jquery is getting used here?

Reply:
pageDown.js is no longer for Angular so used jQuery